stable/database/files/nuobackup

```diff
@@ -161,6 +149,86 @@ function execute_backup() {
    fi
 }
 
+function full_exist() {
+   local error
+
+   error="$( nuodocker get current-backup \
+      --db-name "${db_name}" \
+      --labels "${labels}" 2>&1)"
```

Comment:
Will labels always be passed into this script? I assume so but just checking since this could this fail if they are not

Reply:
The script defaults the labels for `role hotcopy` if none are provided. Please see line 268.